### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by ragecub3**
Turkish language

### CODE REVIEW

File: lang/tr.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+videojs.addLanguage("tr",{
```
Comment (mmcc) [inline lang/tr.json]: This isn't necessary, this file should just be valid JSON.

**Comment by mmcc** [comment]
Thanks for the PR! Fixing my comment should take care of the failing test.

File: lang/tr.json (hunk 1 above)
Comment (ragecub3) [inline lang/tr.json]: Thank you for help. Im new to github :+1:

**Comment by ragecub3** [comment]
Fixed. Works well too.

**Comment by heff** [comment]
looks good to me!

**Comment by heff** [comment]
Thanks @ragecub3!

**Pull request closed without merging**